src/cards.csv

```diff
@@ -135,115 +135,111 @@
-"515","3rd Party services","on which 3rd party does the service rely for operation and what happens if they are down?","","1","","",""
-"516","Personal data","do we deal in any way with personal data?","a","2","","",""
+"515","3rd Party services","Which 3rd part services does our service depend on, and what happens when they do down?","","1","","",""
+"516","Personal data","Do we deal in any way with personal data?","a","2","","",""
 "517","Data Access","Which companies have access to the service data?","","2","","",""
-"518","Business Critical","Is this service business critical and is there a plan if it fails?","","1","","",""
-"519","News","If this service goes banana shaped will it get in the news?","","1","","",""
+"518","Business Critical","Is this service business critical and is there a failover or continuity plan?","","1","","",""
+"519","News","If this service goes pear-shaped will it get in the news?","","1","","",""
 "520","Data","Is data leaving the client domain (3rd party)?","","2","","",""
-"521","System hack","If the system gets hacked in how big trouble are we and is our client?","","1","","",""
-"522","Data integrity","What happens if the data integrity is compromised?","","2","If the data becomes unreliable do we send patients to the wrong operation or is some item temporarily not available on the store. ","",""
-"523","Data availability","What happens if the data availability is interrupted?","","1","Datasource down does that also mean business down?","",""
-"524","Data confidentiality","What happens if the data confidentiality is compromised?","","1","","",""
+"521","System hack","If the system gets hacked what are the potential consequences for our clients and for us?","","1","","",""
+"522","Data integrity","What happens if data integrity is compromised?","","2","If the data becomes unreliable do we send patients to the wrong operation or is some item temporarily not available on the store. ","",""
+"523","Data availability","What happens if data availability is interrupted?","","1","Datasource down does that also mean business down?","",""
+"524","Data confidentiality","What happens if data confidentiality is compromised?","","1","","",""
 "525","Data classification","Is data with different classification treated differently?","","2","thing about a persons social security number vs name of a product in the shop","",""
 "526","Data integrity","How do we know if the data integrety is comprimised?","","2","","",""
-"527","Plain text passwords","Do we log any passwords in pain text?","","2","","",""
-"528","Keys in Github","Do we have any sensitive info in gihub?","","2","Api keys, passwords, etc","",""
+"527","Plain text passwords","Do we log any passwords in plain text?","","2","","",""
+"528","Keys in Github","Do we have any sensitive info in public repositories (Github, etc)?","","2","Api keys, passwords, etc","",""
```

Comment:
scrap public, api keys etc shouldn't be in private repros either

Reply:
👍